.github/actions/run-e2e/action.yaml

```diff
@@ -40,6 +40,15 @@ runs:
       
       mkdir "${ARTIFACTS_DIR}"
       echo "ARTIFACTS_DIR=${ARTIFACTS_DIR}" | tee -a ${GITHUB_ENV}
+  - name: Setup Kubernetes
```

Comment:
which step do you mean exactly? 
I moved it ealier because image loading depends on podman and crictl which config might be changed during kube setup.

Reply:
image and go setup used to be first (the node setup part)

Reply:
rearranged it